scripts/ci/payload-generator/main.go

```diff
@@ -36,40 +39,40 @@ func main() {
 
 func run() error {
 	var (
+		inDir   = defaultInputDir
+		outDir  = defaultOutputDir
 		branch  = constants.BranchName
 		version = constants.Version
 	)
 
+	flag.StringVar(&inDir, "i", inDir, "Override directory to gather payload components from.")
```

Comment:
Once we address my other comment there should be no need anymore for this flag. A big point of the payload-generator is that this logic is centralized. By giving it a custom input path we're just creating more generators in front of generators.